### PR TITLE
 Store plan in local storage 

### PR DESCRIPTION
**Pull request by jeremystucki**

### CODE REVIEW

File: src/views/Home.vue [hunk 1]
```diff
@@ -222,6 +222,12 @@ export default {
               .filter((module) => module),
           }));
       }
+
+      const cachedPlan = localStorage.getItem('plan');
+      if (cachedPlan != null) {
```
Comment (Mafii) [inline src/views/Home.vue:227]: could be a ternary return expression

File: src/views/Home.vue [hunk 2]
```diff
@@ -230,6 +236,10 @@ export default {
         .join('-');
 
       window.location.hash = `plan/${encodedPlan}`;
+
+      if (encodedPlan !== []) {
```
Comment (Mafii) [inline src/views/Home.vue:240]: So the user, when checking out another users plan by opening the url, completely uses his locally saved plan?

**Review by Mafii — CHANGES_REQUESTED** [review]

File: src/views/Home.vue (hunk 2 above)
Comment (jeremystucki) [reply src/views/Home.vue:240]: no, the locally saved one will be replaced. Maybe we need an explicit save button :)

Comment (jeremystucki) [reply src/views/Home.vue:240]: still better to have the behavior introduced in this PR than nothing, so I’d still like to merge it

File: src/views/Home.vue (hunk 1 above)
Comment (jeremystucki) [reply src/views/Home.vue:227]: how? I don’t return within the if

Comment (Mafii) [reply src/views/Home.vue:227]: would work with return cachedPlan != null ? : []

or even

return (window.location.hash = cachedPlan) ?? []

as the return value would correctly be the value of the plan anyways, no? I'm fine with it if you don't care enough but it seems so hacky

File: src/views/Home.vue (hunk 2 above)
Comment (Mafii) [reply src/views/Home.vue:240]: I'd personally rather have a no feature than a fundamentally broken feature, but that's your call.

File: src/views/Home.vue (hunk 1 above)
Comment (jeremystucki) [reply src/views/Home.vue:227]: It didn't work with the second option. No idea why :/

File: src/views/Home.vue (hunk 2 above)
Comment (jeremystucki) [reply src/views/Home.vue:240]: I don't think it's broken. It doesn't break any current user flows and may help some people. Calling it fundamentally broken is a bit much.

**Pull request merged**